src/components/pages/Navbar/Navbar.js

```diff
@@ -20,14 +21,14 @@ const Navbar = ({ isAuthenticated, userProfile }) => {
     <Layout className="layout">
       <Header>
         <div className="logoDiv">
-          <div>
+          <Link to="/dashboard">
```

Comment:
This kind of link is industry standard and will make the app more intuitive.  This is a great addition.